### PR TITLE
Make seperate file workflow_manager file for each module of Swim, Sit…

### PR DESCRIPTION
**Pull request by Abhishek-121**
…e and Inventory, added playbooks for same.

In this commit --

-- Add the module with name - <module_name>_workflow_manager and added same for the playbooks with .py and .yml extension respectively.

-- Added the playbook for all SWIM, Site and Inventory module.

-- Updated class name with component name, replace dnac from function as well with ccc(Cisco Catalyst Center).

-- Remove camel case conversion from SWIM and Site module.

### CODE REVIEW

File: playbooks/inventory_workflow_manager.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        dnac_port: "{{dnac_port}}"
+        dnac_version: "{{dnac_version}}"
+        dnac_debug: "{{dnac_debug}}"
+        dnac_log_level: DEBUG
+        dnac_log: true
+        config_verify: true
+        state: merged
+        config:
+        - username: "{{item.username}}"
+          enable_password: "{{item.enable_password}}"
```
Comment (madhansansel) [inline playbooks/inventory_workflow_manager.yml:25]: password and then enable_password

File: playbooks/inventory_workflow_manager.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          password: "{{item.password}}"
+          ip_address: "{{item.ip_address}}"
+          cli_transport: "{{item.cli_transport}}"
+          # hostname_list: "{{item.hostname_list}}"
```
Comment (madhansansel) [inline playbooks/inventory_workflow_manager.yml:29]: We can remove the commented items or can use it

File: playbooks/site_workflow_manager.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+              area:
+                name: Abc
+                parent_name: 'Global'
+                address: Bengaluru, Karnataka, India
+                latitude: 22.2111
+                longitude: -42.1234434
+                country: "United States"
+            type: area
+
+            # For deleting a site
```
Comment (madhansansel) [inline playbooks/site_workflow_manager.yml:41]: Can you please remove all the commented statements?

File: playbooks/swim_workflow_manager.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        dnac_verify: "{{dnac_verify}}"
+        dnac_port: "{{dnac_port}}"
+        dnac_version: "{{dnac_version}}"
+        dnac_debug: "{{dnac_debug}}"
+        dnac_log: true
+        dnac_log_level: DEBUG
+        config_verify: true
+        config:
+        - import_image_details:
+            # type: "local"
```
Comment (madhansansel) [inline playbooks/swim_workflow_manager.yml:24]: Can you please remove all the commented items?

File: plugins/modules/swim_workflow_manager.py [hunk 5]
```diff
@@ -0,0 +1,1717 @@
...
+    DnacBase,
+    validate_list_of_dicts,
+    get_dict_result,
+)
+from ansible.module_utils.basic import AnsibleModule
+import os
+
+
+class Swim(DnacBase):
+    """Class containing member attributes for Swim intent module"""
```
Comment (madhansansel) [inline plugins/modules/swim_workflow_manager.py:412]: workflow_manager module

File: plugins/modules/site_workflow_manager.py [hunk 6]
```diff
@@ -0,0 +1,1063 @@
...
+    '101101': 'Cubes And Walled Offices',
+    '101102': 'Drywall Office Only',
+    '101105': 'Free Space',
+    '101104': 'Indoor High Ceiling',
+    '101103': 'Outdoor Open Space'
+}
+
+
+class Site(DnacBase):
+    """Class containing member attributes for site intent module"""
```
Comment (madhansansel) [inline plugins/modules/site_workflow_manager.py:326]: workflow_manager module

**Review by madhansansel — APPROVED** [review]

File: plugins/modules/site_workflow_manager.py (hunk 6 above)
Comment (Abhishek-121) [reply plugins/modules/site_workflow_manager.py:326]: added.

File: plugins/modules/swim_workflow_manager.py (hunk 5 above)
Comment (Abhishek-121) [reply plugins/modules/swim_workflow_manager.py:412]: added.

File: playbooks/swim_workflow_manager.yml (hunk 4 above)
Comment (Abhishek-121) [reply playbooks/swim_workflow_manager.yml:24]: removed.

File: playbooks/site_workflow_manager.yml (hunk 3 above)
Comment (Abhishek-121) [reply playbooks/site_workflow_manager.yml:41]: removed.

File: playbooks/inventory_workflow_manager.yml (hunk 2 above)
Comment (Abhishek-121) [reply playbooks/inventory_workflow_manager.yml:29]: we can remove it.

File: playbooks/inventory_workflow_manager.yml (hunk 1 above)
Comment (Abhishek-121) [reply playbooks/inventory_workflow_manager.yml:25]: added in the same order

**Pull request merged**